src/commands/game_commands/leaderboard.ts

```diff
@@ -28,13 +28,19 @@ const logger = new IPCLogger("leaderboard");
 export const ENTRIES_PER_PAGE = 10;
 
 export enum LeaderboardType {
+    EXP = "exp",
+    GAMES_PLAYED = "gamesplayed",
```

Comment:
Can this just be 'songs' and 'games'?

Reply:
I initially considered it, but decided against it because `game` is already an argument

Reply:
We can alternatively separate this into three commands, and maintain the logic in this file